charts/rest-devicemanager/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.0.1
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.1.14"
+appVersion: "0.0.1"
```

Comment:
Yes that was intended.

In the previous version, Alex moved the chart used for testing from `helm-chart-kr` repo.
Since this PR is an intended migration, I restart the appVersion from 0.0.1.